pkg/odo/cli/config/set.go

```diff
@@ -81,6 +82,12 @@ func (o *SetOptions) Complete(name string, cmd *cobra.Command, args []string) (e
 		IsNow:                  o.now,
 		CheckRouteAvailability: checkRouteAvailability,
 	})
+	if err != nil {
+		if util.IsInvalidKubeConfigError(err) {
+			return fmt.Errorf("invalid KUBECONFIG provided. Please point to a valid KUBECONFIG. You do not have to be logged in %w", err)
+		}
+		return err
+	}
```

Comment:
I might be nitpicking, but can we not make the `IsInvalidKubeConfigError` return this statement, so this code block would look like the following?
```go
if err1 := util.IsInvalidKubeConfigError(err); err1 !=nil {
    return err1
}
```

Reply:
updating